{"name": "Unsuccessfully login with email: simeon.hhl.qa@gmail.co and password: Test#1234", "status": "passed", "steps": [{"name": "Check for error", "status": "passed", "start": 1731208916185, "stop": 1731208918441}, {"name": "Check the error text", "status": "passed", "start": 1731208918441, "stop": 1731208918450}], "parameters": [{"name": "email", "value": "'simeon.hhl.qa@gmail.co'"}, {"name": "password", "value": "'Test#1234'"}, {"name": "error", "value": "'There was a problem logging you in. Please try again.'"}, {"name": "driver", "value": "'firefox'"}], "start": 1731208913003, "stop": 1731208918450, "uuid": "ca13357d-bbd4-4596-b44d-b3b7ddd59960", "historyId": "98633738f883cd7872ff013d4e7e2896", "testCaseId": "c2811b1721d8231bbb0fb40778ae099c", "fullName": "Tests.test_Login.TestLogin#test_2", "labels": [{"name": "feature", "value": "Login"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "flaky(reruns=3, reruns_delay=0.5, rerun_except='assert')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az1380-903"}, {"name": "thread", "value": "1746-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}